tests/integrated/test-datafilefacade/runtest

```diff
@@ -0,0 +1,78 @@
+#!/usr/bin/env python3
+#
+# Test file I/O by loading from restart files and writing to dump files
+#
+# requires: netcdf
+# cores: 4
```

Comment:
```suggestion
# cores: 2
```

Reply:
Oh yes, and probably should fix that comment above too